examples/01_plotting/show_ball_sampling.py

```diff
@@ -0,0 +1,72 @@
+"""
+Illustration of the volume to surface sampling scheme
+=====================================================
+
+"""
+
+import numpy as np
+import matplotlib.patches
+from matplotlib import pyplot as plt
+from nilearn.plotting import surf_plotting
+
+
+def _piecewise_const_image():
+    w, h = 7, 10
+    image = np.random.uniform(0, 1, size=(w, h))
```

Comment:
I would like to avoid uncontroled RNGs in the codebase. The right way of doing this is to create a like RNG:
<pre>
rng = np.random.RandomState(0)
image = rng.uniform(...)
</pre>

Reply:
Sorry, I hadn't realized that this was an example, and not a test.

I would simply seed the global RNG here: it's simpler and the side effect is less important.